tools/ci_connector_ops/ci_connector_ops/pipelines/pipelines/metadata.py

```diff
@@ -38,6 +39,29 @@ async def _run(self) -> StepStatus:
         return await super()._run(["metadata_service", "validate", "metadata.yaml"])
 
 
+class DeployOrchestratorStep(Step):
+    def __init__(self, context: PipelineContext, title: str, parent_dir_path: str, module_path: str, dagster_cloud_api_token):
+        self.title = title
+        super().__init__(context)
+        self.parent_dir = self.context.get_repo_dir(parent_dir_path)
+        self.module_path = module_path
+        self.dagster_cloud_api_token = dagster_cloud_api_token
+
+    async def _run(self) -> StepStatus:
+        python_base = with_python_base(self.context)
+        python_with_dependencies = with_pip_packages(python_base, ["dagster-cloud==1.2.6", "poetry2setup"])
+
+        container_to_run = (
+            python_with_dependencies.with_mounted_directory("/src", self.parent_dir)
+            .with_workdir(f"/src/{self.module_path}")
+            .with_env_variable("DAGSTER_CLOUD_API_TOKEN", self.dagster_cloud_api_token)
```

Comment:
```suggestion
            .with_secret_variable("DAGSTER_CLOUD_API_TOKEN", the_token_loaded_in_a_secret_object)
```
Please try to use [`with_secret_variable`](https://dagger-io.readthedocs.io/en/sdk-python-v0.5.0/api.html#dagger.api.gen.Container.with_secret_variable). Here's an [example](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/actions/environments.py#L193) in the connectors pipelines.